torch/_dynamo/optimizations/backends.py

```diff
@@ -133,8 +120,21 @@ def static_runtime(subgraph):
 
 
 def onnxrt_common(subgraph, provider, onnx_filename=None):
+    import numpy as np  # type: ignore[import]
```

Comment:
@ngimel I left this one, as it seems like onnx requires numpy data types, and if this code runs without numpy an import should be thrown similarly to how it would if `onnxruntime` were missing. 

If you wanted this import updated I suppose it would have to throw when `not HAS_NUMPY`

Reply:
Yeah onnx is special, that's fine.